test-fixtures/ci/convert_junit_to_markdown.py

```diff
@@ -223,13 +224,13 @@ def convert_test_cases_to_github_markdown(test_cases, is_smoke = True):
 
 def convert_file_github(input_file, output_file, is_smoke = True):
     test_cases = parse_junit_xml(input_file)
-    markdown = convert_to_github_markdown(test_cases, is_smoke = True)
+    markdown = convert_to_github_markdown(test_cases, is_smoke = is_smoke)
```

Comment:
I forgot to pass the flag so that the full functional test reports were processed as if they were the report from smoke tests.